build_tools/utils.py

```diff
@@ -262,15 +261,15 @@ def copy_common_headers(te_src, dst):
         shutil.copy(file_path, new_path)
 
 
-def install_and_import(package):
-    """Install a package via pip (if not already installed) and import into globals."""
-    main_package = package.split("[")[0]
-    subprocess.check_call([sys.executable, "-m", "pip", "install", package])
-    globals()[main_package] = importlib.import_module(main_package)
+def install_packages(packages):
+    """Install a package via pip (if not already installed)."""
+    for package in packages:
+        main_package = package.split("[")[0]
+    subprocess.run([sys.executable, "-m", "pip", "install", package])
 
 
 def uninstall_te_fw_packages():
-    subprocess.check_call(
+    subprocess.run(
```

Comment:
`check_call` is nice since it will print out the error message if the process fails:
```suggestion
    subprocess.check_call(
```
We could alternatively call with `check=True` or manually check the return code.